src/core/layout/qgslayoutitemlegend.h

```diff
@@ -436,6 +436,13 @@ class CORE_EXPORT QgsLayoutItemLegend : public QgsLayoutItem
 
     void finalizeRestoreFromXml() override;
 
+    /**
```

Comment:
Can you drop the dox from overridden methods? Doxygen automatically handles inheritance of the base class documentation, so we avoid overriding dox as it runs the risk of missing improvements in the base class documentation.